third_party/move/tools/move-mutator/src/output.rs

```diff
@@ -79,10 +71,21 @@ pub(crate) fn setup_mutant_path(
         .ok_or(anyhow::anyhow!("Cannot get file stem of {file_path:?}"))?;
 
     // Deal with the file as OsString to avoid problems with non-UTF8 characters.
-    let mut filename = filename.to_os_string();
-    filename.push(OsString::from(format!("_{index}.move")));
+    let filename = filename.to_os_string();
+    for i in 0u32.. {
+        let mut mutant_path = filename.clone();
+        mutant_path.push(OsString::from(format!("_{i}.move")));
```

Comment:
this would allow quick grepping for mutants (also would make it easier to remove those)
```suggestion
        mutant_path.push(OsString::from(format!("_mut{i}.move")));
```

Reply:
Done.